src/main/java/uk/gov/hmcts/divorce/citizen/notification/conditionalorder/Applicant1ApplyForConditionalOrderNotification.java

```diff
@@ -16,6 +16,10 @@
 @Slf4j
 public class Applicant1ApplyForConditionalOrderNotification {
 
+    public static final String JOINT_CONDITIONAL_ORDER = "Your partner has also received this notification."
```

Comment:
if this is a notificiation template variable, i think it might be best to reduce this and use a short name. as its just going to be replaced.